packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.scss

```diff
@@ -8,4 +8,8 @@
 
 .host {
   display: inline-block;
+}
+
+.hostOverflow {
+  display: inline;
```

Comment:
This change makes me nervous.

Reply:
I don't quite understand this. Is there a better way to fix this?

Reply:
Yes, changing to inline can have some dramatic effects. I'm not too comfortable with this. I really wish the host didn't have to add any DOM (which we can't do until everyone moves to 16.2)

Reply:
It was originally `display: inline` before my PR #3594.